Server/sprawl_calculator/wup_calculator/tasks.py

```diff
@@ -0,0 +1,7 @@
+from celery import Celery
+
+app = Celery('tasks', broker='redis://localhost:8000')
+
+@app.tasks
+def process_algorithm_async(url:str, geojson:GeoJSONFeature):
+    return processAlgorithm(url, geojson)
```

Comment:
Thanks for this, A couple of things: 
- instead of this lets need to copy / paste the code of the process algorithm. 
- In the views.py call this method via the delay() call

Reply:
in the views.py, where to call this ` process_algorithm_async.delay(url, geojson)` or create new functions to call it.
`